apps/openmw/mwmechanics/aipursue.cpp

```diff
@@ -50,8 +50,13 @@ bool AiPursue::execute (const MWWorld::Ptr& actor, CharacterController& characte
 
     //Set the target desition from the actor
     ESM::Pathgrid::Point dest = target.getRefData().getPosition().pos;
+    ESM::Position aPos = actor.getRefData().getPosition();
 
-    if (pathTo(actor, dest, duration, 100)) {
+    float dist = MWMechanics::distance(dest,aPos.pos[0],aPos.pos[1],aPos.pos[2]);
+
+    if (pathTo(actor, dest, duration, 100) &&
+        dist < 100.0)     // check the true distance in case the target is far away in Z direction
```

Comment:
Tried using `MWMechanics::PathFinder::PathTolerance` but it was too small, so I set it to 100.

Reply:
Can we use an activation distance instead (192 by default)?

MWBase::Environment::get().getWorld()->getMaxActivationDistance();
We use this distance in our AI calculations.